packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -6,17 +6,25 @@ import { Localize, localize } from '@deriv/translations';
 import { getCurrencyDisplayCode, getLocalizedBasis, isMobile, getGrowthRatePercentage } from '@deriv/shared';
 import CancelDealInfo from './cancel-deal-info.jsx';
 
-const ValueMovement = ({ has_error_or_not_loaded, proposal_info, currency, has_increased, is_vanilla }) => (
+export const ValueMovement = ({
+    has_error_or_not_loaded,
+    proposal_info,
+    currency,
+    has_increased,
+    is_vanilla,
+    current_value,
```

Comment:
```suggestion
    value,
```
nit :) there is no previous_value or other_value, so we can just make it a `value`